app/src/renderer/components/transactions/TmLiTransaction.vue

```diff
@@ -47,7 +47,10 @@ export default {
   },
   computed: {
     date() {
-      return moment(this.time).format(`MMM Do YYYY, HH:mm:ss`)
+      const time = moment(this.time)
```

Comment:
If you move everything to `this` destructuring should you here also use `date({ time } = this) {`.

Reply:
+ makes the test on this sweetier